Dockerfile

```diff
@@ -6,13 +6,13 @@ FROM tomcat:$IMAGE_VERSION
 LABEL maintainer="Tim Sutton<tim@linfiniti.com>"
 
 ## The Geoserver version
-ARG GS_VERSION=2.15.2
+ARG GS_VERSION=2.16.0
 
 ## Would you like to use Oracle JDK
 ARG ORACLE_JDK=false
 
 ## Would you like to keep default Tomcat webapps
-ARG TOMCAT_EXTRAS=true
+ARG TOMCAT_EXTRAS=false
```

Comment:
@EHJ-52n What is the reason for flagging this off since it is an argument which a user can built with or without

Reply:
okay, let's step back one step before we resolve this.

What are the tomcat extras / default apps?

- manager?
- documentation?
- examples?

If yes, I would recommend to use a tomcat docker image without them by default to save space and do not provide anything that goes beyond having a servlet container with geoserver in it.

What do you think about this?

Reply:
All tomcat docker images have this by default and for us, we have the flag that allows users to built an image with/ without is very nice. We can flag it off by default but we have generally kept it for users who need to administer the servlet through this interface

Reply:
Okay, I will adjust my PR accordingly.